src/pytest_subtests.py

```diff
@@ -176,14 +177,27 @@ def _capturing_output(self):
                 fixture.close()
                 captured.out = out
                 captured.err = err
+    
+    @contextmanager
```

Comment:
Noticing the comment on line 154 above "# capsys or capfd are active, subtest should not capture".  Do I need to do anything similar if the `caplog` fixture is active?

Reply:
I don't think so, the logging capture supports multiple logs being captured at the same time, which is not true for capsys/capfd. :+1: